eng/pipelines/coreclr/templates/build-perf-sample-apps.yml

```diff
@@ -44,6 +44,29 @@ steps:
           archiveExtension: '.tar.gz'
           archiveType: tar
           tarCompression: gz
+    - script: rm -r -f $(Build.SourcesDirectory)/src/mono/sample/iOS/bin
+      workingDirectory: $(Build.SourcesDirectory)/src/mono/sample/iOS
+      displayName: Clean bindir
```

Comment:
I assume this got added because we were seeing some build flakiness?

Reply:
good catch - it's left over from my previous scheme of running the two tests back to back instead of in different runs. Will remove.

Reply:
Oh, no. I was thinking about the wrong place. This is just making sure we are clean to rebuild with LLVM. It stays.